PassiveAgression/PassiveAgression/Characters/ModCompat/Ruina/ArgaliaPassive.cs

```diff
@@ -33,7 +33,7 @@ static ArgaliaPassive(){
          blockPrefab.GetComponentInChildren<EffectComponent>().soundName = "Play_Defense_Guard";
          slot = new CustomPassiveSlot(RiskOfRuinaMod.Modules.Survivors.RedMist.redMistPrefab);
          LanguageAPI.Add("PASSIVEAGRESSION_RUINAARGALIA","Faint Reverberation");
-         LanguageAPI.Add("PASSIVEAGRESSION_RUINAARGALIA_DESC","All <style=cIsDamage>Attack Speed</style> and <style=cIsUtility>Movement Speed</style> bonuses are converted to block chance for long range attacks");
+         LanguageAPI.Add("PASSIVEAGRESSION_RUINAARGALIA_DESC","All <style=cIsDamage>attack speed</style> and <style=cIsUtility>movement speed</style> bonuses are converted to block chance for long range attacks.");
```

Comment:
the [base skill](https://imgur.com/yJk4caG) it's an alt to capitalises them in the same way.

Reply:
vanilla does not capitalize them ... maybe they should fix that one too